components/minio.libsonnet

```diff
@@ -2,15 +2,15 @@
   local minio = self,
 
   config:: {
-    namespace: error 'must provide namespace',
+    s3namespace: error 'must provide namespace',
```

Comment:
we can rename this now that it doesn't clash

Reply:
Done